tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -442,18 +467,19 @@ impl<E> Deref for FormattedFields<E> {
 
 // === impl FmtLayer ===
 
-macro_rules! event_from_span {
-    ($event:ident = $id:ident, $span:ident, $($field:ident = $value:expr),*) => {
+macro_rules! with_event_from_span {
+    ($id:ident, $span:ident, $($field:literal = $value:expr),*, |$event:ident| $code:tt) => {
         let meta = $span.metadata();
         let cs = meta.callsite();
-        let fs = field::FieldSet::new(&[$(stringify!($field)),*], cs);
+        let fs = field::FieldSet::new(&[$($field),*], cs);
```

Comment:
and then, if we apply my suggestion above, we would put back the stringify:
```suggestion
        let fs = field::FieldSet::new(&[$stringify!(($field)),*], cs);
```
again, take it or leave it.